common/lib/calc/setup.py

```diff
@@ -7,6 +7,6 @@
     install_requires=[
         "pyparsing==2.0.1",
         "numpy",
-        "scipy"
+        "scipy<0.18"
```

Comment:
I'm trying out this: pin numpy==1.6.2 and scipy==0.14.0, which are the versions that both edx-platform and ora2 pin.  Is there a reason this won't work?  Where specifically did you find problems?